website/docs/r/compute_cluster_vm_affinity_rule.html.markdown

```diff
@@ -4,41 +4,48 @@ layout: "vsphere"
 page_title: "VMware vSphere: vsphere_compute_cluster_vm_affinity_rule"
 sidebar_current: "docs-vsphere-resource-compute-cluster-vm-affinity-rule"
 description: |-
-  Provides a VMware vSphere cluster virtual machine affinity rule. This can be used to manage rules to tell virtual machines to run on the same host.
+  Provides the VMware vSphere Distributed Resource Scheduler affinity rule resource.
 ---
 
 # vsphere\_compute\_cluster\_vm\_affinity\_rule
 
-The `vsphere_compute_cluster_vm_affinity_rule` resource can be used to manage
-VM affinity rules in a cluster, either created by the
+The `vsphere_compute_cluster_vm_affinity_rule` resource can be used to
+manage virtual machine affinity rules in a cluster, either created by the
 [`vsphere_compute_cluster`][tf-vsphere-cluster-resource] resource or looked up
 by the [`vsphere_compute_cluster`][tf-vsphere-cluster-data-source] data source.
 
+
 [tf-vsphere-cluster-resource]: /docs/providers/vsphere/r/compute_cluster.html
 [tf-vsphere-cluster-data-source]: /docs/providers/vsphere/d/compute_cluster.html
 
-This rule can be used to tell a set to virtual machines to run together on a
-single host within a cluster. When configured, DRS will make a best effort to
+This rule can be used to tell a set to virtual machines to run together on the
+same host within a cluster. When configured, DRS will make a best effort to
 ensure that the virtual machines run on the same host, or prevent any operation
 that would keep that from happening, depending on the value of the
 [`mandatory`](#mandatory) flag.
 
--> Keep in mind that this rule can only be used to tell VMs to run together on
-a _non-specific_ host - it can't be used to pin VMs to a host. For that, see
-the
+An affinity rule places a group of virtual machines on the same 
```

Comment:
```suggestion
```

Reply:
This seemed like a duplicated paragraph, I would have liked to suggest a multiline delete but I'm not sure GitHub can do that? Or if you can I don't know how.